tests/report_tests/smoke_tests.py

```diff
@@ -97,20 +94,25 @@ def test_rhel_default_plugins(self):
         """Plugins expected to always run on a RHEL (-like) system
         """
         self.assertPluginIncluded([
-            'anaconda',
```

Comment:
Interesting, I as under the impression this is a must.. But I can easily get e.g. a RHEL system where the plugin is not enabled.